apps/studio/components/interfaces/App/FeaturePreview/FeaturePreview.constants.tsx

```diff
@@ -1,6 +1,13 @@
 import { LOCAL_STORAGE_KEYS } from 'common'
 
 export const FEATURE_PREVIEWS = [
+  {
+    key: LOCAL_STORAGE_KEYS.UI_PREVIEW_SECURITY_NOTIFICATIONS,
+    name: 'Security notification templates',
+    discussionsUrl: undefined,
+    isNew: true,
+    isPlatformOnly: false,
```

Comment:
Should `isPlatformOnly` be `true`? My understanding of `platform` is that it means the “Supabase-hosted version” of Studio. Does that mean these email templates only apply when that’s the case?

CC @issuedat

Reply:
^ yup! for self-hosted / local, there's no dashboard support for emails so this is correct